tests/components/controlBar-test.js

```diff
@@ -112,7 +112,136 @@ describe('ControlBar', function () {
     TestUtils.Simulate.click(fullscreenButton);
     expect(fullscreenToggled).toBe(true);
   });
+  
+  it('render one stereo button if content vr', function () {
+    var mockController = {
```

Comment:
Please use `baseMockController` and `baseMockProps` properties instead of declaring new ones each time.

Reply:
Now this part of the code is removed from the userstory. We add props to baseMockController in userstory 1424. Thank you.